doc/user_guide/dynamo0p3.rst

```diff
@@ -2037,7 +2037,11 @@ following rules:
    ``integer`` scalar arguments;
 
 7) Built-ins that update ``integer``-valued fields can only read from
-   other ``integer``-valued fields and take ``integer`` scalar arguments.
+   other ``integer``-valued fields and take ``integer`` scalar arguments
+
+.. note:: The exceptions from taking only the fields of the same data
```

Comment:
I don't think there's a need to a note on an exception that doesn't exist (yet).

Reply:
Ok, removed.